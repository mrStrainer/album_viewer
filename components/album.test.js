import React from 'react';
import Album from './album';
import AlbumHeader from './albumHeader';
import FetchError from './fetchError';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<Album />).toJSON();
  expect(rendered).toMatchSnapshot();
});

it('renders without crashing', () => {
  const rendered = renderer.create(<AlbumHeader />).toJSON();
  expect(rendered).toMatchSnapshot();
});

it('renders without crashing', () => {
  const rendered = renderer.create(<FetchError />).toJSON();
  expect(rendered).toMatchSnapshot();
});