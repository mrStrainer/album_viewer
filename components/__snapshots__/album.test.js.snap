// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;

exports[`renders without crashing 2`] = `
<View>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "height": 300,
        "marginBottom": 10,
        "position": "relative",
        "width": 300,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#000",
        "bottom": 10,
        "opacity": 0.5,
        "padding": 4,
        "position": "absolute",
        "width": 300,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "textAlign": "center",
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ccc",
          "textAlign": "center",
        }
      }
    />
  </View>
</View>
`;

exports[`renders without crashing 3`] = `null`;
